packages/node-sdk/README.md

```diff
@@ -315,7 +315,22 @@ the number of calls that are sent to Bucket's servers. During process shutdown,
 some messages could be waiting to be sent, and thus, would be discarded if the
 buffer is not flushed.
 
-A naive example:
+By default, the SDK automatically subscribes to process exit signals and attempts to flush
```

Comment:
above we say

> It is highly recommended that users of this SDK manually call `flush()`
method on process shutdown.

i don't think we need to recommend that anymore. This auto-flush should "just work".